compile_protobuf.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Standard Library
+import glob
+import logging
+import os
+import platform
+import shutil
+import sys
+import tempfile
+import time
+import urllib
+import urllib.request
+from subprocess import check_call
+from zipfile import ZipFile
+
+
+def script_name() -> str:
+    """:returns: script name with leading paths removed"""
+    return os.path.split(sys.argv[0])[1]
+
+
+def configure_logging():
```

Comment:
nit: `setup.py` uses print statements and we use logging here. To be more consistent, can we make everything logging statements (preferable) or print statements?